### PR TITLE
SAK-45052 Dashboard and Tests & Quizzes: Always create a task item when publishing assessments.

### PR DESCRIPTION
**Pull request by druiz309**
The specs on this Jira never mentioned for conditional creation of tasks on the dashboard. Consequently, radio buttons and some methods have been removed. Currently, tests are automatically created, updated and removed. Anyone can also see tests in the tasks widget in the dashboard.

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/PublishAssessmentListener.java [hunk 1]
```diff
@@ -100,252 +100,262 @@
 import org.springframework.web.client.HttpClientErrorException;
 
 /**
- * <p>Title: Samigo</p>2
- * <p>Description: Sakai Assessment Manager</p>
+ * <p>
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/PublishAssessmentListener.java:103]: Revert these changes

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/PublishAssessmentListener.java [hunk 2]
```diff
@@ -100,252 +100,262 @@
...
+ * Description: Sakai Assessment Manager</p>
+ *
  * @author Ed Smiley
  * @version $Id$
  */
 @Slf4j
 public class PublishAssessmentListener
-    implements ActionListener {
+        implements ActionListener {
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/PublishAssessmentListener.java:113]: Revert these changes

**Review by mpellicer — CHANGES_REQUESTED** [review]
Revert format changes please, focus only on removing the checkbox associated to the task.

**Review by mpellicer — CHANGES_REQUESTED** [review]

**Comment by druiz309** [comment]
I have just made the changes @mpellicer

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Good job @druiz309 !

**Comment by mpellicer** [comment]
@druiz309 Could you change the commit message to this? 

'SAK-45052 Dashboard and Tests & Quizzes: Always create a task item when publishing assessments.'

**Comment by mpellicer** [comment]
I believe we will recover these changes in the future if the spec advances, I like the idea of controlling the task item by a checkbox but at the same time is more work for the instructor, as it's not clear in the specs what the community wants let's simplify the use cases.

**Comment by druiz309** [comment]
> @druiz309 Could you change the commit message to this?
> 
> 'SAK-45052 Dashboard and Tests & Quizzes: Always create a task item when publishing assessments.'

I have changed the commit message @mpellicer

**Review by mpellicer — APPROVED** [review]

**Pull request merged**